docker/README.md

```diff
@@ -13,17 +13,17 @@ Requires a running deployment of registry
 
 To build and run  (assuming registry local-dev defaults for host/credentials)
 
-    cd path/to/registry-sweepers
-    docker build -t registry-sweepers .
+    cd path/to/registry-sweepers/
+    docker build -t registry-sweepers docker --file ./docker/Dockerfile .
```

Comment:
I think there's an extra `docker` appearing in this command line.

Also, when writing documentation, I like to see the long form of arguments because I think they're more "documentary" 😇

So I'd make this `docker image build --tag registry-sweepers --file ./docker/Dockerfile .`

Reply:
ok